### DISCUSSION TITLE
почему жалуется вирустотол?

### DISCUSSION THREAD

**Discussion by sanatosss** [hufrea/byedpi#51] [General]
почему жалуется вирустотол?

**Comment by sanatosss** [comment]
@
![Picsart_24-08-08_17-01-49-104](https://github.com/user-attachments/assets/7cd35dc2-bc4a-4188-8c26-1b9692950682)

**Reply by lufog** [reply]
Всегда можно:
1. Самостоятельно проинспектировать открытый код. 
2. Собрать самому программу из этого кода. 
3. Убедиться что False Detection второе имя большинства антивирусов.

А если это претензия к Android клиенту, то ей занимается другой человек, в другом репозитории. К слову он там отвечал из-за чего происходит False Detection.

**Reply by lufog** [reply]
Ну да, как я и предполагал, все чисто
Linux: https://www.virustotal.com/gui/file/20abf3ec48798b1a17de55b9ca8c70a4ac270e8bf58725d45d680de8efb4c40f
Windows: https://www.virustotal.com/gui/file/14f6d465b78d8679b2dd3481f20e2a5089b9b45306bc68a915d44b397801db0d

**Reply by sanatosss** [reply]
андроид клиент

**Reply by lufog** [reply]
Вы не по адресу, им занимается другой человек. Вам нужно [сюда](https://github.com/dovecoteescapee/ByeDPIAndroid/issues/54).

**Reply by sanatosss** [reply]
большое спасибо за наводку <3
